wordpress-mobile/gutenberg-mobile
# Fix Aztec Android "SetSpan" ends beyond length on merging blocks

This PR does fix a sub-case of #872 and wordpress-mobile/WordPress-Android#9832 where the editor crashes on merging 2 blocks when the first block does have BR tag(s) at the end of its content.

GB side PR: https://github.com/WordPress/gutenberg/pull/18138

To test:
**Note, we should test this PR with tons of input since the changes in Aztec may break other part of the GB-Editor.**
- Create a post on Gutenberg web with a para block that ends with one or more empty line(s) (Shift+Enter)
![Screenshot 2019-10-23 at 19 55 08](https://user-images.githubusercontent.com/518232/67420522-0b695a80-f5cf-11e9-8719-5289d97034ec.png)
- Open this new post in GB mobile
- Tap on the `Plus` symbol to add a new block below the para
- Put the focus into this new block and taps on Backspace to merge this new block with the one above.
- It should not crash, and the 2 blocks merged, with the caret at the end of the block

Also try many other testing scenarios.


cc @SergioEstevao 

Update release notes:

- [ x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
